test/integration/targets/yum/tasks/yum.yml

```diff
@@ -427,3 +427,114 @@
     that:
         - "'changed' in yum_result"
         - "'msg' in yum_result"
+
+# setup for testing installing an RPM from url
+- name: remove sos
+  yum:
+    name: sos
+    state: absent
```

Comment:
Should it also user pkg_name (and so define it before ?)

Reply:
Fixed, thanks.